packages/browser/src/index.ts

```diff
@@ -88,6 +93,75 @@ export default class LogtoClient {
     sessionStorage.setItem(this.logtoStorageKey, jsonItem);
   }
 
+  public async getAccessToken(resource?: string): Promise<Optional<string>> {
+    if (!this.idToken) {
+      throw new LogtoClientError('not_authenticated');
+    }
+
+    const accessTokenKey = buildAccessTokenKey(resource);
+    const accessToken = this.accessTokenMap.get(accessTokenKey);
+
+    if (!accessToken) {
+      throw new LogtoClientError('get_access_token_failed');
+    }
+
+    if (accessToken.expiresAt < Date.now()) {
+      this.accessTokenMap.delete(accessTokenKey);
+
+      const { clientId, requester } = this.logtoConfig;
+      const { tokenEndpoint } = await this.getOidcConfig();
```

Comment:
just want to double-check will getOidcConfig throws if no proper config is fetched?

Reply:
Yes, it will throw errors

Reply:
Then we can just remove try catch in this function if we're going to naturally throw these errors. Or we wrap all of them into one error scope. Say `get_access_token.unable_fetch_oidc_config`

Reply:
Yeah, I see your points. After checking the code, I found `tokenEndpoint` is only needed in the try-catch block, so I moved it to the `try` block and if `getOidcConfig` is failed, the error will be wrapped into the same error scope: `get_access_token_by_refresh_token_failed`.

Does this sound good to you? @gao-sun @simeng-li